spec/helper.js

```diff
@@ -21,6 +21,27 @@ global.onHttpContext = index.onHttpContextFactory();
 // Legacy
 global.dihelper = onHttpContext.helper;
 
+helper.httpServerBefore = function(overrides, endpointOpt) {
+    before('helper.httpServer.before', function() {
+        this.timeout(5000);
+        return helper.startServer(overrides, endpointOpt);
+    });
+
+    beforeEach(function () {
```

Comment:
I think your idea is calling
```
helper.before();
helper.httpServerBefore();
```
instead of
```
helper.httpServerBefore();
```
But we don't need `helper.start`  in `on-http`, so none of the unit test files calls `helper.before`.